### ISSUE TITLE
Issue with Stack which uses Java _and_ Python lambdas

### ISSUE THREAD

**Issue by nicklester** [cfn-modules/lambda-function#40]
We have an application stack using Java/Scala and Python lambdas. Only problem is, this means the python source ends up in the same lambda-src directory as the java (which is significant in package size).

Is there a way we can keep these two frameworks separate?

**Comment by michaelwittig** [comment]
Unfortunately, no. You could split it into two projects if that is possible.

**Comment by nicklester** [comment]
Ok - thanks. Judging by your comment in module.yml, does this mean it is an issue even if we used cloudformation without the Lambda template module?

**Comment by michaelwittig** [comment]
@nicklester No, I don't think so. The issue is that `aws cloudformation package` (the piece that uploads the cfn template and lambda-src) does not support parameters. Therefore, we can not make the folder `lambda-src` configurable.
